desktop/src/renderer/src/components/transfer/UploadGrid/DeleteCell.tsx

```diff
@@ -0,0 +1,35 @@
+import { IconButton, Tooltip } from "@mui/material";
+import { DeleteOutline as DeleteIcon } from "@mui/icons-material";
+import type {
+  GridRenderCellParams,
+  GridTreeNodeWithRender,
+} from "@mui/x-data-grid";
+import type { Row } from "./FolderUploadGrid";
+
+type Props = {
+  params: GridRenderCellParams<Row, unknown, unknown, GridTreeNodeWithRender>;
+  onFolderDelete: (folder: string) => Promise<void> | void;
+};
+
+export const DeleteCell = ({ params, onFolderDelete }: Props) => {
+  return (
+    <Tooltip
+      title="Delete folder from transfer."
+      sx={{
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+        width: "100%",
+        height: "100%",
+      }}
+    >
+      <IconButton
```

Comment:
Not an issue - `DeleteCell` component is already fairly clean and minimalistic 💯 

Just one thought maybe for future to-do: `IconButton` has a consistent pattern with `EditCell` components, we could consider reusing a `CellActionButton` component to handle similar `button` elements across different actions like edit, delete, etc.

Reply:
Yeah could be an idea for the future